web/channels/game_channel.ex

```diff
@@ -8,7 +8,8 @@ defmodule Reactor.GameChannel do
   end
 
   def broadcast_winner(topic, %{user: user}) do
-    Reactor.Endpoint.broadcast("game:#{topic}", "new:winner", %{user: user})
+    {:ok, users} = GameManager.get_users(topic)
```

Comment:
I'm not sure what this should be called. In Channel terms they call it a topic but it's also the game_id